packages/mds-utils/utils.ts

```diff
@@ -508,13 +498,27 @@ function getPolygon(geographies: Geography[], geography: string): Geometry | Fea
 }
 
 function isInStatesOrEvents(rule: Rule, event: VehicleEvent): boolean {
-  const status = rule.states ? rule.states[EVENT_STATES_MAP[event.event_type] as VEHICLE_STATE] : null
-  return status !== null
-    ? rule.states !== null &&
-        Object.keys(rule.states).includes(EVENT_STATES_MAP[event.event_type]) &&
-        status !== undefined &&
-        (status.length === 0 || (status as string[]).includes(event.event_type))
-    : true
+  const { states } = rule
+  // If no states are specified, then the rule applies to all VehicleStates.
+  if (states === null || states === undefined) {
+    return true
+  }
+
+  // States that it is possible to transition into with event.event_type
+  const possibleStates: string[] = EVENT_STATES_MAP[event.event_type]
```

Comment:
The definition of `VehicleEvent` needs to be updated, event-types come in as an Array in 1.0.0, not a singleton.